das/sign_after_store_das.go

```diff
@@ -154,10 +155,12 @@ func (d *SignAfterStoreDAS) Store(ctx context.Context, message []byte, timeout u
 		}
 	}
 
-	c = &arbstate.DataAvailabilityCertificate{}
-	copy(c.DataHash[:], crypto.Keccak256(message))
+	c = &arbstate.DataAvailabilityCertificate{
+		Timeout:  timeout,
+		DataHash: dastree.Hash(message),
+		Version:  1,
+	}
```

Comment:
Incorporate the `c.SignersMask = 1` assignment into here.

Reply:
Incorporated!